packages/shorebird_cli/lib/src/commands/release/release_android_command.dart

```diff
@@ -139,162 +140,165 @@ Use `shorebird flutter versions list` to list available versions.
       flutterRevisionForRelease = revision;
     }
 
-    final originalFlutterRevision = shorebirdEnv.flutterRevision;
-    final switchFlutterRevision =
-        flutterRevisionForRelease != originalFlutterRevision;
-
-    if (switchFlutterRevision) {
-      await shorebirdFlutter.useRevision(revision: flutterRevisionForRelease);
-    }
-
-    final flutterVersionString = await shorebirdFlutter.getVersionAndRevision();
-
-    final buildProgress = logger.progress(
-      'Building release with Flutter $flutterVersionString',
+    final flutterInstallProgress = logger.progress(
+      'Installing Flutter $flutterRevisionForRelease',
```

Comment:
So do we always show this log even when the revision of flutter is already installed?

Reply:
Yep, but I'll fix that and add error handling for install failures. One sec.

Reply:
Updated